torch/testing/_internal/common_methods_invocations.py

```diff
@@ -350,6 +350,30 @@ def sample_inputs_broadcast_to(op_info, device, dtype, requires_grad):
                                           requires_grad=requires_grad), shape))
                  for size, shape in test_cases)
 
+def sample_inputs_div(self, device, dtype, requires_grad):
+    a = make_tensor((S, S, S), device, dtype, low=None, high=None, requires_grad=requires_grad)
```

Comment:
It's unfortunate that make_tensor doesn't currently around for two separate domains to pull values from. For types that support negative values we should add a second "b" tensor to pair a with that contains negative values.